client/views/msg_action.tpl

```diff
@@ -1,4 +1,4 @@
-<div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}" id="msg-{{id}}" data-time="{{time}}">
+<div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}{{#if sameSender}} equal-from{{/if}}" id="msg-{{id}}" data-time="{{time}}">
```

Comment:
I don't think we want that on actions, what's the use case? "Condensing" senders between normal messages and action messages would be unfortunate.